guides/common/modules/proc_installing-capsule-server-packages.adoc

```diff
@@ -28,4 +37,27 @@ ifndef::satellite[]
 ----
 # {package-install} foreman-proxy-content
 ----
+
+ifdef::foreman-el,katello[]
+== [[centos-7-package]]CentOS 7
+
+:package-manager: yum
+include::proc_installing-proxy-packages-el.adoc[]
+
+endif::[]
+
+ifdef::foreman-el,katello[]
+== [[centos-8-package]]CentOS 8
```

Comment:
Don't we support RHEL 8 upstream?

Reply:
I'm going to assume yes unless anyone else says otherwise.